src/panel/common/commonActions.js

```diff
@@ -79,10 +79,10 @@ function isInvalidTags(tagArr) {
 function isInvalidId(id) {
 	// Checks whether id is a number and an integer
 	if (Number.isInteger(parseFloat(id))) {
```

Comment:
The issue with this method is that `1.` as an ID is accepted by the method, but rejected by the server. This could potentially be remedied by converting the ID to an integer before sending it to the server. This would then entail converting the ID to an integer if it is valid according to the method above, and then convert back to string